frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -559,6 +560,13 @@ export const supportLogic = kea<supportLogicType>([
                 }
             }
 
+            const startupProgramLabel = billing?.startup_program_label || billing?.startup_program_label_previous
+            if (startupProgramLabel === StartupProgramLabel.YC) {
+                planLevelTag = 'plan_yc'
+            } else if (startupProgramLabel === StartupProgramLabel.Startup) {
+                planLevelTag = 'plan_startup'
+            }
+
```

Comment:
Might be worth having a different tag for current vs previous startup/yc program members? AFAIK the support tier and SLAs would differ depending on that.

Reply:
> Once they're off of Startup or YC, they should be tagged with whatever their new plan is, shouldn't they?
I think so too.

> Or will the startup/yc-ness persist somehow even after they're no longer on it?
That's more of a question to you - would it impact things from support perspective? Would we handle them differently? Probably not.

The reason why I flagged this was due to this line

```
const startupProgramLabel = billing?.startup_program_label || billing?.startup_program_label_previous
```

Right now, after their startup/YC program ends, they're still going to have the `plan_yc` or `plan_startup` label - not sure if we want this since it might wrongly impact the SLAs etc?